CONVERTER-SERVICE/src/converter/video.converter.js

```diff
@@ -0,0 +1,38 @@
+/*
+@Video.converter.js Copyright (c) 2022 Jalasoft
+CI 26 Sur #48-41, Ayurá Center, Edificio Unión № 1376, Medellín, Colombia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const FfmpegCommand = require('fluent-ffmpeg');
+
+//This class represents a model for convert Video into Images 
+class VideoConverter {
+
+  //This method allows to convert a video into an specif number of images by setting the Frames Per Second with an specific size 
```

Comment:
it should start in third person with a verb:
for example: //Allows to convert a video into an specif number of images by setting the Frames Per Second with an specific size

Reply:
You are right! I already corrected those comments! Thanks!